src/merkle/smt/full/mod.rs

```diff
@@ -344,6 +406,30 @@ impl SparseMerkleTree<SMT_DEPTH> for Smt {
     fn path_and_leaf_to_opening(path: MerklePath, leaf: SmtLeaf) -> SmtProof {
         SmtProof::new_unchecked(path, leaf)
     }
+
+    fn pairs_to_leaf(mut pairs: Vec<(RpoDigest, Word)>) -> SmtLeaf {
+        assert!(!pairs.is_empty());
+
+        // FIXME
+        if cfg!(debug_assertions) {
+            let mut control = pairs.clone();
+            control.sort_by_key(|(key, _)| Self::key_to_leaf_index(key).index.value());
+            assert_eq!(control, pairs);
+        }
```

Comment:
Is this temporary code?

Reply:
Yes! Thank you; I thought I caught all of those.